src/main/java/seedu/duke/logic/commands/circuit/TutorialCircuitCommand.java

```diff
@@ -11,7 +11,8 @@
 
 public class TutorialCircuitCommand extends SummaryCommand implements TutorialCommand {
     private final Parser parser = new Parser();
-    private static final String[] orderOfInstructions = {"template", "set v", "set", "set", "add", "calc"};
+    private static final String[] orderOfInstructions = {"template rc", "set v", "set r", "set c", "add parallel c",
```

Comment:
Yup try maybe add wrong commands not matching these in input.txt to see what happens.

Reply:
Okie cool